src/components/subpages/parallel-sign-in-hello.tsx

```diff
@@ -23,17 +23,53 @@ export const ParallelSignInHello: React.FC<Props> = (props) => {
         client: { isAuthenticated, username },
     } = useSelector((state: RootState) => state.user);
 
+    // State to handle dynamic rendering
+    const [clientRendered, setClientRendered] = React.useState(false);
+
+    React.useEffect(() => {
+        // Set client-rendered state after component mounts
+        setClientRendered(true);
+    }, []);
+
+    if (!clientRendered) {
+        return null;
+    }
```

Comment:
Why is this necessary?

Reply:
There's a bug where the format is very strange if you access the page directly URL. This workaround was already done for the L2 subpage but not yet for the parallel subpage. So in the current deployment, you can see the bug if you go directly to https://samromur.is/parallel